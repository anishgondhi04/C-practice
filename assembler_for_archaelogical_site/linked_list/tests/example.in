2
list 0 new
list 1 new
list 0 build 5 ali ben gem dan eve
list 1 build 5 gem dan eve ali ben 
list 0 overlap 1
list 0 splice 1
list 1 print
end

