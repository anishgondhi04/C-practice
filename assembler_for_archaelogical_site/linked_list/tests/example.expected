List 0: new list
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 0: build
 [ ali ben gem dan eve ] (5 items)
List 1: build
 [ gem dan eve ali ben ] (5 items)
List 0: Overlap between lists 0 and 1 is 3
List 0: List 1 spliced to the end of list 0
 [ ali ben gem dan eve ali ben ] (7 items)
List 1: print
 [ gem dan eve ] (3 items)
